packages/appeals-service-api/src/controllers/confirm-email.js

```diff
@@ -16,7 +16,8 @@ async function confirmEmailCreate(req, res) {
 		res.status(400).send('Invalid Id');
 		throw new Error('');
 	}
-	const saved = await confirmEmailCreateService(appeal);
+	const savedAppealId = await confirmEmailCreateService(appeal);
+	const saved = {appealId: savedAppealId}
 	await confirmEmailNotifyContinue(appeal);
```

Comment:
i don't think the unit test for this function is testing the res.send - perhaps a good idea to update?

Reply:
Added test